viz/js/scaiiViz.js

```diff
@@ -465,6 +476,18 @@ function handleViz(vizData) {
   renderState(masterEntities);
 
 }
+function buildEchoVizInitMultiMessage(vizInit) {
+  var returnScaiiPacket = new proto.scaii.common.ScaiiPacket;
+  returnScaiiPacket.setVizInit(vizInit);
+  var mm = buildReturnMultiMessageFromScaiiPacket(returnScaiiPacket);
+  return mm;
+}
+function buildMultiMessageWithUserCommand(userCommand) {
```

Comment:
Can you add a whitespace line between your function definitions? The lack of whitespace makes it a little hard to read this file. (Maybe leave this for a separate commit later, doesn't have to be right now).

Reply:
certainly